packages/documentation-framework/components/example/example.css

```
@@ -101,32 +67,32 @@
 }
 
 .pf-v6-c-badge.ws-beta-badge {
-/*   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-v6-global--BackgroundColor--100); */
-/*   --pf-v6-c-badge--m-unread--Color: var(--pf-v6-global--primary-color--100); */
-/*   border: var(--pf-v6-global--BorderWidth--sm) solid var(--pf-v6-global--primary-color--100); */
+   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-t--global--background--color--floating--default);
+   --pf-v6-c-badge--m-unread--Color: var(--pf-t--global--color--status--unread--default--default);
+   border: var(--pf-t--global--border--width--regular) solid var(--pf-t--global--border--color--nonstatus--blue--default);
 }
 
 .ws-prop-required {
-/*   color: var(--pf-v6-global--danger-color--100); */
+  color: var(--pf-t--global--text--color--status--danger--default);
 }
 
 .ws-full-page-utils {
   position: fixed;
   right: 0;
   bottom: 0;
   padding: var(--pf-t--global--spacer--lg);
-/*   z-index: var(--pf-v6-global--ZIndex--2xl); */
+  z-index: var(--pf-t--global--z-index--600);
 }
 
 .ws-full-page-utils::before {
   position: absolute;
   inset: 0;
   content: "";
-/*   background-color: var(--pf-v6-global--BackgroundColor--100); */
+  background-color: var(--pf-t--global--background--color--floating--default);
   opacity: 0.8;
-/*   box-shadow: var(--pf-v6-global--BoxShadow--sm); */
+  box-shadow: var(--pf-t--global--box-shadow--color--sm);
```

box-shadow--color--sm is just a color. Box shadows are another where it's a 1:1 replacement with the new token syntax... with the exception of `pf-v5-global--BoxShadow--inset` and `pf-v5-global--BoxShadow--xl`, which don't exist in v6.

```suggestion
  box-shadow: var(--pf-t--global--box-shadow--color--sm);
```